### PR TITLE
Add gitpod to Adopters 🎉

### PR DESCRIPTION
**Pull request by ArthurSens**
**Release Note Template (will be copied)**

```release-note:REPLACEME

```

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -106,6 +106,19 @@ Details (optional):
 - A huge fleet of OpenShift and Kubernetes clusters, each using Prometheus Operator
 - All managed by [Project Syn](https://syn.tools/), leveraging Commodore Components like [component-rancher-monitoring](https://github.com/projectsyn/component-rancher-monitoring) which re-uses Prometheus Operator
 
+
+## Gitpod
```
Comment (lilic) [inline ADOPTERS.md:110]: This file is in alphabetical order, do you mind just recording it, thanks! 🎉

**Review by lilic — COMMENTED** [review]
Thank you!! 🙇‍♀️ 🎉 

One small nit, we had requests from folks to reorder this before so trying to keep it ordered.

**Comment by ArthurSens** [comment]
Oh, I didn't notice that, fixed!

**Comment by paulfantom** [comment]
Thanks! :+1: 

---

I am merging this even though e2e tests are failing as this is only documentation.

**Pull request merged**